web/src/condense.ts

```diff
@@ -262,14 +282,21 @@ export function condense_and_collapse(elems) {
     }
 }
 
-export function initialize() {
+export function initialize(): void {
     $("#message_feed_container").on("click", ".message_expander", function (e) {
         // Expanding a message can mean either uncollapsing or
         // uncondensing it.
         const $row = $(this).closest(".message_row");
         const id = rows.id($row);
+        if (id === undefined) {
+            return;
+        }
         assert(message_lists.current !== undefined);
         const message = message_lists.current.get(id);
+        if (message === undefined) {
+            return;
+        }
```

Comment:
These should be assert statements, not `if` statements; the `message_expander` element can only be present on message rows that actually exist in those data structures.